src/Symfony/Component/Validator/Tests/Mapping/Loader/PropertyInfoLoaderTest.php

```diff
@@ -58,11 +59,27 @@ public function testLoadClassMetadata()
                 [new Type(Type::BUILTIN_TYPE_FLOAT, true)], // The existing constraint is float
                 [new Type(Type::BUILTIN_TYPE_STRING, true)],
                 [new Type(Type::BUILTIN_TYPE_STRING, true)],
-                [new Type(Type::BUILTIN_TYPE_ARRAY, true, null, true, null, new Type(Type::BUILTIN_TYPE_FLOAT))]
+                [new Type(Type::BUILTIN_TYPE_ARRAY, true, null, true, null, new Type(Type::BUILTIN_TYPE_FLOAT))],
+                [new Type(Type::BUILTIN_TYPE_STRING)]
+            ))
+        ;
+        $propertyInfoStub
+            ->method('isWritable')
+            ->will($this->onConsecutiveCalls(
+                true,
+                true,
+                true,
+                true,
+                true,
+                true,
+                true,
+                true,
+                true,
+                false
```

Comment:
It would be clearer with a `@dataProvider` or splitting these tests into different ones.

Reply:
A data provider will not help. We could indeed split in in several tests, I'm not sure that is really worth it, but anyway can it be done in another PR because it's not related to this fix.